week-3/C-array-every/exercise.js

```diff
@@ -6,12 +6,15 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 
 // -- Complete this function -->
-function hasOnlyStudents() {}
+function hasOnlyStudents(studentName) {
```

Comment:
How is this function used below (line 14). Is it passed a single `studentName`?. Given how it is used does the logic here make sense.

Reply:
My understanding here:
the group [ ], must be defined before using it. in this case it was declared on line 14 and I used evey() method and called my function right after line 14.

Reply:
My mistake I think. I was looking at the original line 14 which calls the function passing in `groups`. If you rewrite it this way though you should rename the function `hasOnlyStudents` to something more appropriate (like `isStudent(name)`) since it is again only checking a single value. The original version was intended to check whether a group of names "has only students". This one checks whether a name "is a student".